TIP-0003/README.md

```diff
@@ -0,0 +1,273 @@
+
+##### Attributes
+
+| Attribute Name     | Type                   | Description                                                  |
+| ------------------ | ---------------------- | ------------------------------------------------------------ |
+| seriesId           | Hash32                 | The series identifier of this series constructor token. It is the digest of the Series Policy. |
+| quantity           | Int128                 | The quantity of series constructor tokens stored in a given UTXO. |
+| tokenSupply        | Int [ 0 .. 1 ]         | This is an optional field. When provided it fixes the quantity of tokens that can be minted when this series is consumed, and the series constructor is burned by the minting transaction. When not provided, the series constructor is not burned, thus making the token supply unlimited. |
+| fungibility        | FungibilityType        | Describes the fungibility of the assets minted using this series constructor token. |
+| quantityDescriptor | QuantityDescriptorType | Describes the behavior of the quantity field of the assets minted using the series constructor derived from this policy. |
+
+#### Group Policy
+
+The group policy describes a group of tokens and the behavior of a Group Constructor Token when used in a minting transaction.
+
+##### Attributes
+
+| Attribute Name   | Type             | Description                                                  |
+| ---------------- | ---------------- | ------------------------------------------------------------ |
+| label            | String           | The human readable name of this group.                       |
+| registrationUtxo | UtxoAddress      | The address of a UTXO. The UTXO contains the LVLs that are paid for minting the group constructor token. |
+| fixedSeries      | Hash32 [ 0 .. 1] | This is an optional field. When provided, it means that this group can only be used to mint an asset with the series that has the same identifier as this field. |
+
+#### Series Policy
+
+The series policy describes a series of tokens and the behavior of a Series Constructor Token when used in a minting transaction.
+
+##### Attributes
+
+| Attribute Name          | Type                   | Description                                                  |
+| ----------------------- | ---------------------- | ------------------------------------------------------------ |
+| label                   | String                 | The human readable name of this series.                      |
+| registrationUtxo        | UtxoAddress            | The address of a UTXO. The UTXO contains the LVLs that are paid for minting the group constructor token. |

```

Comment:
The address of a UTXO. The UTXO contains the LVLs that are paid for minting the **series** constructor token.